pip/pip-272.md

```diff
@@ -0,0 +1,108 @@
+# Background knowledge
+
+In Pulsar, a pulsar function support storing state, such as a `WordCount` function which stores the state of its counters.
+
+```python
+from pulsar import Function
+
+class WordCount(Function):
+    def process(self, item, context):
+        for word in item.split():
+            context.incr_counter(word, 1)
+```
+
+Currently, Pulsar uses Bookkeeper as the default state storage interface. We can also use other state stores, which can be configured in the `conf/functions_worker.yml` using the field: `stateStorageProviderImplementation`, this YAML file will be parsed and loaded in Pulsar as the `WorkerConfig`.
+
+## WorkerConfig
```

Comment:
@asafm I added `functions_worker.yml` related info in here

Reply:
Looks good

Reply:
Then shall we restart the voting process?